### PR TITLE
Fixing triage button transition

### PR DESCRIPTION
**Pull request by kevgliss**
When the triage button is clicked we need to set triage_at.

### CODE REVIEW

File: src/dispatch/plugins/dispatch_slack/case/interactive.py [hunk 1]
```diff
@@ -1245,6 +1245,14 @@ def triage_button_click(
     case = case_service.get(db_session=db_session, case_id=context["subject"].id)
     case.status = CaseStatus.triage
     db_session.commit()
+    # we run the case status transition flow
+    case_flows.case_status_transition_flow_dispatcher(
+        case=case,
+        current_status=CaseStatus.triage,
+        db_session=db_session,
+        previous_status=case.status,
```
Comment (wssheldon) [inline src/dispatch/plugins/dispatch_slack/case/interactive.py:1253]: Would this be overwritten by the line above it?

```python
    case.status = CaseStatus.triage
    db_session.commit()
```

So, it's always `triage` -> `triage`

Comment (kevgliss) [reply src/dispatch/plugins/dispatch_slack/case/interactive.py:1253]: Ah, yeah let me change the order.

**Review by wssheldon — APPROVED** [review]

File: src/dispatch/plugins/dispatch_slack/case/interactive.py [hunk 2]
```diff
@@ -1242,6 +1242,14 @@ def triage_button_click(
...
     ack()
+    # we run the case status transition flow
+    case_flows.case_status_transition_flow_dispatcher(
+        case=case,
+        current_status=CaseStatus.triage,
+        db_session=db_session,
+        previous_status=case.status,
+        organization_slug=context["subject"].organization_slug,
+    )
     case = case_service.get(db_session=db_session, case_id=context["subject"].id)
```
Comment (wssheldon) [inline src/dispatch/plugins/dispatch_slack/case/interactive.py:1253]: Need to move this up below `ack()`. Thank you ruff🙏

File: src/dispatch/plugins/dispatch_slack/case/interactive.py [hunk 3]
```diff
@@ -1242,6 +1242,14 @@ def triage_button_click(
     ack: Ack, body: dict, db_session: Session, context: BoltContext, client: WebClient
 ):
     ack()
```
Comment (wssheldon) [inline src/dispatch/plugins/dispatch_slack/case/interactive.py:1244]:
```suggestion
    ack()
    
    case = case_service.get(db_session=db_session, case_id=context["subject"].id)
```

**Pull request merged**